src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -0,0 +1,189 @@
+            self.assertEqual(device['cbsdCategory'], cbsd_record[0]['cbsdCategory'])
+            self.assertEqual(device['fccId'], cbsd_record[0]['fccId'])
+            self.assertEqual(device['airInterface']['radioTechnology'],\
+                             cbsd_record[0]['airInterface']['radioTechnology'])
+            self.assertEqual(device['installationParam']['latitude'],\
+                             cbsd_record[0]['installationParam']['latitude'])
+            self.assertEqual(device['installationParam']['longitude'],\
+                             cbsd_record[0]['installationParam']['longitude'])
+            self.assertEqual(device['installationParam']['height'],\
+                             cbsd_record[0]['installationParam']['height'])
+            self.assertEqual(device['installationParam']['heightType'], \
+                             cbsd_record[0]['installationParam']['heightType'])
+            self.assertEqual(device['installationParam']['indoorDeployment'], \
+                             cbsd_record[0]['installationParam']['indoorDeployment'])
+            self.assertEqual(device['installationParam']['antennaGain'], \
+                             cbsd_record[0]['installationParam']['antennaGain'])     
+            self.assertEqual(device['installationParam']['antennaAzimuth'], \
+                             cbsd_record[0]['installationParam']['antennaAzimuth'])        
+            self.assertEqual(device['installationParam']['antennaDowntilt'], \
+                             cbsd_record[0]['installationParam']['antennaDowntilt'])       
+            self.assertEqual(device['installationParam']['antennaBeamwidth'], \
+                             cbsd_record[0]['installationParam']['antennaBeamwidth'])    
+                         
+            # Get grants by cbsd_id
+            grants_of_cbsd = [cbsd['grants'] for cbsd in recordData if cbsd['id'] == record_id]
+            self.assertEqual(1, len(grants_of_cbsd))
+            self.assertTrue('id' in grants_of_cbsd[0])
+            # Verify the Grant Of the Cbsd
+            if grants_of_cbsd[0]['requestedOperationParam'] is not None :
+                self.assertEqual(grants_of_cbsd[0]['requestedOperationParam']['maxEirp'],\
+                             grant_request[index]['operationParam']['maxEirp'])    
+                self.assertEqual(grants_of_cbsd[0]['requestedOperationParam']['operationFrequencyRange']\
+                                 ['lowFrequency'], grant_request[index]['operationParam']['operationFrequencyRange']['lowFrequency'])
+                self.assertEqual( grants_of_cbsd[0]['requestedOperationParam']\
+                                 ['operationFrequencyRange']['highFrequency'], grant_request[index]['operationParam']\
+                                 ['operationFrequencyRange']['highFrequency']) 
+                
+            self.assertEqual( grants_of_cbsd[0]['operationParam']['maxEirp'],\
```

Comment:
You might be able to do a dictionary comparison of `grants_of_cbsd[0]['operationParam']` and `grant_request[index]['operationParam']` here and save yourself some trouble.